javaparser-core/src/main/java/com/github/javaparser/utils/Log.java

```diff
@@ -47,7 +47,7 @@ private void printStackTrace(Throwable throwable) {
                 throwable.printStackTrace(pw);
                 trace(sw::toString);
             } catch (IOException e) {
-                throw new AssertionError("Error in logging library");
+                throw new AssertionError("Error in logging library", e);
```

Comment:
This is supposed to be unreachable code, but if it *is* reached it might be good to know exactly what happened - so this one is useful :-)